client.go

```diff
@@ -689,19 +692,21 @@ func newLink(s *Session, r *Receiver, opts []LinkOption) (*link, error) {
 		MaxMessageSize:     l.maxMessageSize,
 	}
 
+	attach.Source = l.source
+	attach.Target = l.target
```

Comment:
Can you move these into the struct literal above?

``` go
	attach := &performAttach{
		Name:               l.name,
		Handle:             l.handle,
		ReceiverSettleMode: l.receiverSettleMode,
		SenderSettleMode:   l.senderSettleMode,
		MaxMessageSize:     l.maxMessageSize,
		Source:             l.source,
		Target:             l.target,
	}
```

Reply:
Sure, done.